apps/web/src/components/auth/forgot-form.tsx

```diff
@@ -20,15 +21,28 @@ export default function ForgotForm() {
         },
     });
 
+    async function forgotPasswordCallback(data: z.infer<typeof forgotSchema>) {
+        let res: DefaultActionReturn = undefined;
+        try {
+            res = await forgotPassword(data);
+        } catch (err) {
+            throw new Error("Ein Fehler ist aufgetreten.");
+        }
+
+        if (res && !res?.success) {
+            throw new Error("");
```

Comment:
Vllt sollte hier dann auch eine genaue Fehlermeldung mit in den Error gegeben werden
```suggestion
            throw new Error(res.message);
```